assets/multiqc_config.yml

```diff
@@ -0,0 +1,282 @@
+    pct_adapter: 300
+    pct_surviving: 310
+    pct_duplication: 320
+    after_filtering_gc_content: 330
+    after_filtering_q30_rate: 340
+    after_filtering_q30_bases: 350
+    filtering_result_passed_filter_reads: 360
+  Adapter Removal:
+    aligned_total: 360
+    percent_aligned: 370
+    percent_collapsed: 380
+    percent_discarded: 390
+  Porechop:
+    Input Reads: 400
+    Start Trimmed: 410
+    Start Trimmed Percent: 420
+    End Trimmed: 430
+    End Trimmed Percent: 440
+    Middle Split: 450
+    Middle Split Percent: 460
+  Filtlong:
+    Target bases: 500
+  BBDuk:
+    Input reads: 800
+    Total Removed bases percent: 810
+    Total Removed bases: 820
+    Total Removed reads percent: 830
+    Total Removed reads: 840
+  PRINSEQ++:
+    prinseqplusplus_total: 900
+  bowtie2:
+    overall_alignment_rate: 1000
+  Samtools Stats:
+    raw_total_sequences: 1100
+    reads_mapped: 1110
+    reads_mapped_percent: 1120
+    reads_properly_paired_percent: 1130
+    non-primary_alignments: 1140
+    reads_MQ0_percent: 1150
+    error_rate: 1160
+  Bracken:
+    "% Unclassified": 1200
+    "% Top 5": 1210
+  Centrifuge:
+    "% Unclassified": 1300
+    "% Top 5": 1310
+  DIAMOND:
+    queries_aligned: 1400
+  Kaiju:
+    assigned: 1500
+    "% Assigned": 1510
+    "% Unclassified": 1520
+  Kraken:
+    "% Unclassified": 1600
+    "% Top 5": 1610
+  MALT:
+    "Num. of queries": 1700
+    Total reads: 1710
+    Mappability: 1720
+    Assig. Taxonomy: 1730
+    Taxonomic assignment success: 1740
+  motus:
+    Total number of reads: 1800
+    Number of reads after filtering: 1810
+    Total number of inserts: 1820
+    Unique mappers: 1830
+    Multiple mappers: 1840
+    Ignored multiple mapper without unique hit: 1850
+    "Number of ref-mOTUs": 1860
+    "Number of meta-mOTUs": 1870
+    "Number of ext-mOTUs": 1880
+
+table_columns_visible:
+  FastQC / Falco (pre-Trimming):
+    total_sequences: True
+    avg_sequence_length: True
+    percent_duplicates: True
+    percent_gc: True
+    percent_fails: False
+  FastQC / Falco (post-Trimming):
+    total_sequences: True
+    avg_sequence_length: True
+    percent_duplicates: False
+    percent_gc: False
+    percent_fails: False
+  porechop:
+    Input reads: False
+    Start Trimmed:
+    Start Trimmed Percent: True
+    End Trimmed: False
+    End Trimmed Percent: True
+    Middle Split: False
+    Middle Split Percent: True
+  fastp:
+    pct_adapter: True
+    pct_surviving: True
+    pct_duplication: False
+    after_filtering_gc_content: False
+    after_filtering_q30_rate: False
+    after_filtering_q30_bases: False
+  Filtlong:
+    Target bases: True
+  Adapter Removal:
+    aligned_total: True
+    percent_aligned: True
+    percent_collapsed: True
+    percent_discarded: False
+  BBDuk:
+    Input reads: False
+    Total Removed bases Percent: False
+    Total Removed bases: False
+    Total Removed reads percent: True
+    Total Removed reads: False
+  "PRINSEQ++":
```

Comment:
I am no mqc config expert, so just double checking that the quotes here don't do anything unexpected, since it is the only tool that has them?

Reply:
Nope - should be fine, it just allows the `++` to be included in the section name :)